src/com/amazon/kinesis/streaming/agent/tailing/AbstractParser.java

```diff
@@ -172,6 +175,7 @@ private boolean setCurrentFile(TrackedFile file, boolean resetParsing) {
                 currentFileChannel = currentFile.getChannel();
                 currentFileChannelOffset = currentFileChannel.position();
                 headerLinesToSkip = currentFileChannelOffset == 0 ? flow.getSkipHeaderLines() : 0;
+                fileFooterPattern = flow.getFleFooterPattern();
```

Comment:
type here, should be getFileFooterPattern(). I guess that's why it was failing

Reply:
Will fix, thanks